# 开始建图服务
# 请求
bool need_calibration    # 是否需要校准雷达
---
# 响应
bool success             # 操作是否成功
string message           # 返回消息


